### PR TITLE
ci: Reintroduce redis services at 6379

### PR DESCRIPTION
**Pull request by achals**
Signed-off-by: Achal Shah <achals@gmail.com>

**What this PR does / why we need it**:
We moved this to a github action in #2350 and #2354 but this change wasn't needed. I'd like to move the tests back to using the standard redis port so that the tests can be run while developing locally without any changes.

Corresponding test changes to use this redis instance and remove the old one will be in a followup PR.

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: .github/workflows/master_only.yml [hunk 1]
```diff
@@ -66,6 +66,16 @@ jobs:
     env:
       OS: ${{ matrix.os }}
       PYTHON: ${{ matrix.python-version }}
+    services:
```
Comment (felixwang9817) [inline .github/workflows/master_only.yml:69]: I think you missed a supercharge below, line 132

**Review by felixwang9817 — CHANGES_REQUESTED** [review]
missed a supercharge

File: .github/workflows/master_only.yml (hunk 1 above)
Comment (achals) [reply .github/workflows/master_only.yml:69]: I'm doing to remove this the superchange actions after this PR gets in.

**Review by felixwang9817 — APPROVED** [review]
/lgtm

**Pull request merged**